doc/whats_new.rst

```diff
@@ -0,0 +1,44 @@
+:orphan:
+
+.. include:: links.inc
+
+.. _whats_new:
+
+.. currentmodule:: spykes
+
+.. _current:
+
+Current
+-------
+
+Changelog
+~~~~~~~~~
+
+    - ...
+
+BUG
+~~~
+
+    - ...
+
+API
+~~~
+
+    - ...
+
+
+Version 0.1
+-----------
+
+Changelog
+~~~~~~~~~
+
+    - Add NeuroVis class by `Hugo Fernandes`_ and `Pavan Ramkumar`_
```

Comment:
can you say ``:class:`pyglmnet.NeuroVis``, like here: https://github.com/mne-tools/mne-python/blob/master/doc/whats_new.rst